app/app.js

```diff
@@ -1,4 +1,5 @@
 var allChickens = [];
+var view ={};
```

Comment:
While you obviously can put all this code in one file, it doesn't make it easier to read, manage, maintain. Haven't we reviewed MVC concepts? Better practice is to have separate files for the Model and View components.